lib/messagebird.js

```diff
@@ -1028,7 +1037,148 @@ module.exports = function (accessKey, timeout, features) {
         httpRequest({ method: 'DELETE', path: '/groups/' + groupId + '/contacts/' + contactId }, callback);
       }
 
-    }
+    },
+    calls: {
+       /**
+       * Create a call.
+       *
+       * @param {Object} params
+       * @param {Function} callback
+       * @return void
+       */
+      create: function (params, callback) {
+        httpRequest(
+          {
+            hostname: VOICE_ENDPOINT,
+            method: 'POST',
+            path: '/calls',
+            params,
+          },
+          callback,
+        );
+      },
+      /**
+       * List calls.
+       *
+       * @param {Function} callback
+       * @return void
+       */
+      list: function (callback) {
+        httpRequest(
+          {
+            hostname: VOICE_ENDPOINT,
+            method: 'GET',
+            path: '/calls',
+          },
+          callback,
+        );
+      },
+       /**
+       * Read a call.
+       *
+       * @param {String} callId
+       * @param {Function} callback
+       * @return void
+       */
+      read: function (callId, callback) {
+        httpRequest(
+          {
+            hostname: VOICE_ENDPOINT,
+            method: 'GET',
+            path: `/calls/${callId}`,
+          },
+          callback,
+        );
+      },
+      /**
+       * Delete a call.
+       *
+       * @param {String} callId
+       * @param {Function} callback
+       * @return void
+       */
+      delete: function (callId, callback) {
+        httpRequest(
+          {
+            hostname: VOICE_ENDPOINT,
+            method: 'DELETE',
+            path: `/calls/${callId}`,
+          },
+          callback,
+        );
+      },
+    },
+    recordings: {
+      /**
+       * View an existing recording.
+       *
+       * @param {String} callId
+       * @param {String} legId
+       * @param {String} recordingId
+       * @param {Function} callback
+       * @return void
+       */
+      read: function (callId, legId, recordingId, callback) {
+        httpRequest(
+          {
+            hostname: VOICE_ENDPOINT,
+            method: 'GET',
+            path: `/calls/${callId}/legs/${legId}/recordings/${recordingId}`,
+          },
+          callback,
+        );
+      },
 
+      /**
+       * List recordings.
+       *
+       * @param {String} callId
+       * @param {String} legId
+       * @param {Function} callback
+       * @return void
+       */
+      list: function (callId, legId, limit, offset, callback) {
```

Comment:
How would you feel about using an options object like so:

```
recordings.list({ callId: '123', legId: '456', limit: 5, offset: 0 })
```

I think it leads to a bit more clarity in the consuming code because you'll see what the values actually mean instead of seeing this: 
```
recordings.list('123', '456', 5, 0)
```

Reply:
Sure it does make sense, I just didn't want to change a pattern that is used for other methods, that all the parameters pass separately. So usage of the recordings list method and for example, the contacts list method will be the same.

Reply:
Fair point!